### PR TITLE
Updated to penguinV namespace

### PR DESCRIPTION
**Pull request by RahulRavishankar**
relates to #543

### CODE REVIEW

File: src/cuda/image_buffer_cuda.cuh [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #include "../image_buffer.h"
 #include "cuda_device.cuh"
 
-namespace PenguinV_Image
+namespace PenguinV
```
Comment (ihhub) [inline src/cuda/image_buffer_cuda.cuh]: @RahulRavishankar could you please rename into `penguinv` without capital **P** everywhere?

**Comment by ihhub** [comment]
Hi @RahulRavishankar there are a plenty of files which are failed to be compiled. You could look into https://travis-ci.org/ihhub/penguinV/jobs/591624755 for example. We have many files which use Image classes.

File: src/cuda/image_buffer_cuda.cuh (hunk 1 above)
Comment (RahulRavishankar) [reply src/cuda/image_buffer_cuda.cuh]: Yes!

File: src/penguinv/penguinv.h [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
 namespace penguinV
 {
-    using namespace PenguinV_Image;
+
```
Comment (ihhub) [inline src/penguinv/penguinv.h]: Could you please remove these 2 empty lines.

**Comment by ihhub** [comment]
hi @RahulRavishankar could you please take a look [this compilation log](https://travis-ci.org/ihhub/penguinV/jobs/592037139) to see missing files which need to be changed.

**Comment by ihhub** [comment]
Closing this pull request as author is not responding anymore.

**Pull request closed without merging**